internal/html/form.go

```diff
@@ -107,7 +116,10 @@ var formSpecSnippet = NewSnippet(`
 	<form method="POST" action={{.Spec.PostTarget}}>
 		{{.Fields}}
 		<div class="button-row">
-			<button type="submit" class="button button-primary">{{.Spec.SubmitLabel}}</button>
+			{{- range .Links }}
+			<a class="button button-primary" href="{{.Target}}" >{{.DisplayName}}</a>
+			{{- end }}
+			<button type="submit" class="button button-{{if .Warning}}warning{{else}}primary{{end}}">{{.Spec.SubmitLabel}}</button>
```

Comment:
I couldn't find a button-error, so I just used the warning for disabling TOTP again. Maybe you can add that to the css?